app/pages/project/classify.cjsx

```diff
@@ -230,7 +230,7 @@ module.exports = React.createClass
 
   saveClassificationAndLoadAnotherSubject: ->
     @saveClassification()
-      .then @loadAnotherSubject()
+    @loadAnotherSubject()
 
```

Comment:
[Line 241](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/project/classify.cjsx#L241) stores the classification in a local variable within the method, which should be okay since that won't be changed by loading a new subject and updating state.

If we add arguments to `saveClassification`, we'll have to find everything that invokes it as a callback, and ensure that those calls pass in a classification too.